frontend/src/pages/PipelineDetails.tsx

```diff
@@ -523,10 +520,7 @@ class PipelineDetails extends Page<{}, PipelineDetailsState> {
           logger.error('Cannot retrieve pipeline versions.', err);
           return;
         }
-        templateString = await this._getTemplateString(
-          pipelineId,
-          v2SelectedVersion ? v2SelectedVersion.pipeline_version_id! : v1SelectedVersion?.id!,
-        );
```

Comment:
Can we remove `v1SelectedVersion` as well? Just curious if we can rely on v2 API as much as we can.

Reply:
Plan to do it in next PR if it's workable : )